djautotask/sync.py

```diff
@@ -1575,6 +1579,42 @@ def _assign_field_data(self, instance, object_data):
         return instance
 
 
+class AttachmentSynchronizer:
+    client_class = api.AttachmentInfoAPIClient
+
+    def __init__(self, type, field=None, *args, **kwargs):
```

Comment:
Should type be a keyword argument here too?

Reply:
Yes.